src/azuredb/dbconn.js

```diff
@@ -0,0 +1,3 @@
+const storage = require('azure-storage');
```

Comment:
can be moved. The file is created not for the purpose of getting json from azure blob. It was a test for azure storage table